website/4x4.html

```diff
@@ -123,9 +123,14 @@ <h2>Unused characters:</h2>
     On subsequent runs, the modules it uses will already have been transpiled.
     It will go fast and any message you change will flash quickly.
     -->
-    <script type="text/javascript"> update_loader_message("Loading Program Code....") </script>
+    <script type="text/javascript"> update_loader_message("Loading Program Code...") </script>
     <!-- Boilerplate code starts from here -->
 
+    <script>
+      update_loader_message("Loading Voice...");
+      var voices = speechSynthesis.getVoices();
```

Comment:
Good job on investigating and finding those StackOverflow hints. Looks like [this suggestion](https://stackoverflow.com/a/22978802) makes sense. We can discuss (offline) to see what to do here.